### ISSUE TITLE
Update an existing import

### ISSUE THREAD

**Issue by jusabatier** [public-transport/gtfs-via-postgres#37]
I succed import a dataset in my PostgreSQL database.

But now, I wonder how I can update it if the data changes ?
When I try to re-import the data it crash as the tables already exists and can't be created again.

Should I try do delete and recreate the entire schema before import ?

**Comment by derhuerst** [comment]
In order to stay simple, `gtfs-via-postgres` assumes that an import of a GTFS dataset is immutable, i.e. will never be updated.

What do you actually want to achieve?

If you want to import new data next to the old data, I suggest to
- either import into a different schema (e.g. the date instead of `public`) using `--schema` (this allows you to compare data by running queries across both schemas),
- or import the data into a separate database, e.g. using `CREATE DATABASE <date_of_new_dataset>` & the `PGDATABASE=<date_of_new_dataset>` env variable (this gives you two entirely separate "buckets" of data, of which you can delete individual ones in a straightforward way (`DROP DATABASE <name>`).

**Comment by jusabatier** [comment]
In our usecase, the GTFS is generated from the software of the service which manage the transport lines.
So if they edit them, the exported GTFS will change, and we want to have an up-to-date version in the postgresql database.

We don't have to keep old versions in our database, only the actual one.

So I think I will try to delete schema if exists before import as it seems to be the more simple solution for this usecase.

**Comment by derhuerst** [comment]
Yes, that seems to solve the use case. I think using separate database is the cleaner solution though.

In case you *do* want a "zero downtime update", you can import the new data into a new database next to the old one, and only delete the old one after you have switched your business logic (that accesses the data in the database) to the new database.

**Comment by derhuerst** [comment]
Just a reminder: `gtfs-via-postgres` is [licensed](https://github.com/public-transport/gtfs-via-postgres/blob/5d00586b2bb942912a5ef9e3d4d2952e8ce15edf/readme.md#license) so that personal/non-commercial use is allowed, but commercial organisations (with some exceptions) will have to pay for it after a testing period.

**Comment by jusabatier** [comment]
As I work for an administration, I think we're in the non-commercial use case ?

**Comment by jusabatier** [comment]
I work directly for it and it aims to provide informations about transports on the collectivity's website to citizen.
